corona-ide-core/src/main/java/com/coronaide/core/service/impl/ProjectService.java

```diff
@@ -98,21 +92,21 @@ public Project create(ProjectRequest request) throws IOException {
 
         datastoreService.store(workspace, module, datastore, new WorkspaceMetaData(locations));
 
-        return toProject(projectLocation, coreConfiguration.getWorkingDirectoryName());
+        return toProject(projectLocation);
     }
 
     @Override
     public Collection<Project> getAll() {
         Workspace workspace = workspaceService.getActiveWorkspace();
         Module module = CoronaIdeCore.getModule();
-        Datastore<WorkspaceMetaData> datastore = Datastores.getWorkspaceDatastore();
-        Path workingDirectoryName = coreConfiguration.getWorkingDirectoryName();
+        Datastore<WorkspaceMetaData> datastore = com.coronaide.core.internal.datastore.impl.Datastores
```

Comment:
That's next - the fully-qualified name Datastores will become CoreDatastores in the API project